livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java

```diff
@@ -353,7 +351,7 @@ public void testSparkRInterpreter() {
     // TODO(zjffdu),  Livy's SparkRIntepreter has some issue, do it after livy-0.3 release.
   }
 
-  @Test
+//  @Test
```

Comment:
Need to disable it as it could cause OOM in CI due to docker container memory limitation